upgrade_test.py

```diff
@@ -1076,16 +1076,10 @@ def test_kubernetes_platform_upgrade(self):
         self.monitors.reconfigure_scylla_monitoring()
         self.k8s_cluster.check_scylla_cluster_sa_annotations()
 
-        InfoEvent(message='Step9 - Verify data in the Scylla cluster').publish()
+        InfoEvent(message=f"Step{next(step)} - Verify data in the Scylla cluster").publish()
+        self.verify_stress_thread(stress_thread_pool)
         self.fill_and_verify_db_data(note='after operator upgrade and scylla member addition')
 
-        InfoEvent(message='Step10 - Run c-s read workload').publish()
```

Comment:
why are you remove the read after the upgrade finished ? no need to validate that anymore ?

Reply:
The stress gets validated on the new line 1080 after long rung during the upgrade.

Reply:
I think there is some value in running a read only load, after the whole upgrade is done, since where reading during we can miss data getting lost, since we read it before the upgrade, but not after.

I would strive to be as close as possible to the upgrade test accumulated knowledge, 
for example, initially we didn't add load during the upgrade (not sure, what were the reasons), just to later figure out how impotent that was.

Reply:
I am not sure I understood you.

What we have in this PR now:
1) We run only "read" kind of the load.
2) We verify the whole data set in the end of the test. It takes about 4 hours having 40Gb of data.
And the point (2), I think, does exactly what you say (as I understood) - re-read whole data set and verify it.

Or if to rephrase it then we have here following:
- Write 40Gb of data in the beginning of the test
- Read these data during the upgrade (possibly we read just part of it)
- Read the whole data set and verify it in the end of the test.

So, I don't see any problem in the current approach.

Reply:
Old line 1086 is superseded with the new line 1080. And the new one verify whole data set, not small part as it was before.

Reply:
`verify_stress_thread()` basically  waits for stress to finish, that the stress that started before the upgrade, if i'm not mistaken

in step10, it was starting a new stress, and reading the whole dataset from start (or at least what was configured in `stress_cmd_r`)

Reply:
I'm still trying to understand why we are removing the full read after the upgrade...